src/main/python/scripts/remove_duplicate_links.py

```diff
@@ -0,0 +1,76 @@
+import sys
+import shapefile
+
+if len(sys.argv) != 3:
+    print("python remove_duplicate_links.py <input shapefile> <output location>")
+    exit()
+
+reader = shapefile.Reader(sys.argv[1])
+writer = shapefile.Writer(sys.argv[2])
+
+# START
+# accumulate adjacent links max_speed
+inbound_links = dict()
+outbound_links = dict()
+
+def adjacent_links(d, key, value):
+    if key not in d:
+        d[key] = set([value])
+    else:
+        s = d[key]
+        s.add(value)
+
+for record in reader.iterRecords():
+    [from_node, to_node] = record['ID'].split('-')
```

Comment:
is it possible there will be more than two pieces after the split?

Reply:
No. 

Note: the script is specifically written for Seattle emme shapefile.

Reply:
But I'll add safe guard in subsequent commit. Thanks.

Reply:
But on the second thought, lets keep as it is. Becuase if developer will try to use the script other than Seattle emme shapefiles, they will get notified with "not enough (or too many) value to unpack" and they can decide what chages they want to make. Thoughts? @nikolayilyin